hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/exec/v049/Version049FeatureFlags.java

```diff
@@ -0,0 +1,29 @@
+/*
```

Comment:
Was it necessary to create this subclass?  Could we not just use the `v046` one?

Reply:
placeholder in case we need it for something?  easier to do both these classes, side-by-side, at once?  easier for someone to read the code and see a bunch of `v049` in the source code without finding a `v046` and wondering what's up with that?

I don't feel strongly about it.  I could remove it.

Reply:
I don't feel strongly one way or the other either, just curious.